AnkiDroid/src/main/java/com/ichi2/anki/IntentHandler.kt

```diff
@@ -375,5 +381,19 @@ class IntentHandler : AbstractIntentHandler() {
                     ).toLong()
             }
         }
+
+        /**
+         * Returns an intent to review a specific deck.
+         * This does not states which reviewer to use, instead IntentHandler will choose whether to use the
+         * legacy or the new reviewer.
+         * It is expected to be used from widget, shortcut, reminders but not from ankidroid directly because of the CLEAR_TOP flag.
+         * In this name, "shortucts" should be understood as anything that skip the deck picker.
```

Comment:
```suggestion
         * "shortucts" should be understood as anything that skips the deck picker.
```
or remove this line. Idk if this helps much